src/main/java/hudson/plugins/ec2/util/MinimumInstanceChecker.java

```diff
@@ -0,0 +1,42 @@
+package hudson.plugins.ec2.util;
+
+import hudson.plugins.ec2.EC2Cloud;
+import hudson.plugins.ec2.EC2Computer;
+import hudson.plugins.ec2.SlaveTemplate;
+import jenkins.model.Jenkins;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class MinimumInstanceChecker {
+
+  public static int countCurrentNumberOfSlaves(@Nonnull SlaveTemplate slaveTemplate) {
+    return (int) Arrays.stream(Jenkins.get().getComputers()).filter(computer -> {

```

Comment:
Nit: I'd prefer if it was 4 spaces like the rest of the project

Reply:
I agree. Just the default settings from my IDE that came through. Fixed!